reporter/console.go

```diff
@@ -0,0 +1,37 @@
+package reporter
+
+import (
+	"log"
+	"strconv"
+	"sync"
+
+	"github.com/wavefronthq/wavefront-opentracing-sdk-go/tracer"
+)
+
+// ConsoleSpanReporter send spand to STDOUT.
+type ConsoleSpanReporter struct {
+	sync.Mutex
+}
+
+// NewConsoleSpanReporter returns a ConsoleSpanReporter.
+func NewConsoleSpanReporter() tracer.SpanReporter {
+	return &ConsoleSpanReporter{}
+}
+
+// ReportSpan complies with the SpanReporter interface.
+func (r *ConsoleSpanReporter) ReportSpan(span tracer.RawSpan) {
+	tags := prepareTags(span)
+	parents, followsFrom := prepareReferences(span)
+
+	r.Lock()
+	defer r.Unlock()
+
+	log.Printf("-- Operation: %v\n", span.Operation)
```

Comment:
You could call https://github.com/wavefrontHQ/wavefront-sdk-go/blob/master/senders/formatter.go#L108 and print the output. Will help us debug issues down the line if we can see it in our line format.